image-engine/src/main/java/com/safehill/kcrypto/image_engine/model/ImageFilterWorkState.kt

```diff
@@ -0,0 +1,24 @@
+package com.safehill.kcrypto.image_engine.model
+
+import android.net.Uri
+
+sealed interface ImageFilterWorkState {
+
+    data class Running(
+        val progress: Float,
```

Comment:
Who is in charge of updating this progress?

Reply:
ImageFilterWorker takes care of reporting progress